xsec/plotCrossSectionFromFile.C

```diff
@@ -47,10 +47,10 @@ void SetPOT(TFile& fin, CCPi::MacroUtil& util) {
 //==============================================================================
```

Comment:
These changes don't need to be committed.

Smaller pull requests/branches are better.